rwengine/src/render/VisualFX.hpp

```diff
@@ -50,7 +50,7 @@ struct ParticleFX final : public VisualFX {
     float lifetime{-1.f};
 
     /** Texture name */
-    TextureData::Handle texture;
+    TextureData* texturePtr = nullptr;
```

Comment:
nit: I don't really like the Hungarian notation (using -Ptr as suffix).
I don't think it is used (much) elsewhere.

Reply:
Should be ok now.